Devscord.DiscordFramework/Middlewares/Contexts/Permissions.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Devscord.DiscordFramework.Commons;
+
+namespace Devscord.DiscordFramework.Middlewares.Contexts
+{
+    public class Permissions : ICollection<Permission>
+    {
+        private readonly ICollection<Permission> _permissions;
+        public ulong RawValue => (ulong)_permissions.Sum(x => (long)x);
```

Comment:
tak sie zastanawiam czy nie lepiej byłoby zrobić tego jako ExtensionMethod  

internal static ulong GetRawValue<T>(this IEnumerable<T> permissions) where T : Permission
w commons dla frameworka 

jakieś PermissionsExtensions

Reply:
W czym by to pomogło? Obecnie chyba nie przeszkadza czytelności, bo jest krótkie i proste

Reply:
bardziej chodzi o to, że mamy dodatkową klase Permissions 
której prawie cały kod to przekazywanie do ICollection które trzyma w sobie